charts/castai-hosted-model/templates/placement-job.yaml

```diff
@@ -23,6 +23,15 @@ spec:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
               - matchExpressions:
+                  {{- if gt (len .Values.placementJob.blacklistedGPUNames) 0 }}
+                  - key: nvidia.com/gpu.name
+                    operator: NotIn
+                    values:
+                    {{- $blacklist := required "placementJob.blacklistedGPUNames is required" .Values.placementJob.blacklistedGPUNames }}
```

Comment:
We check that the blacklistedGPUNames length is higher than 0, so I assume this `required` validation is irrelevant?